src/main/java/org/jenkinsci/test/acceptance/recorder/TestRecorderRule.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2014 Red Hat, Inc.
```

Comment:
Most probably it is a part implemented by @olivergondza . Personally I would vote for "Jenkins developers" for a highly shared code, but each company has its own policy.